public/app/features/alerting/unified/rule-list/components/RuleGroupActionsMenu.tsx

```diff
@@ -0,0 +1,22 @@
+import { Dropdown, IconButton, Menu } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+export function RuleGroupActionsMenu() {
```

Comment:
(From a design point of view) There's a lot of empty space in the right of the group item, perhaps we can work out a way to display more information while balancing overcrowding the page, with hiding things away too much

Reply:
Agreed, we'll cover some of those defects with Deyan and the rest of the team once we've got these functional changes in.